src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.Session_ComputeModel.cs

```diff
@@ -177,12 +185,7 @@ private static bool DisplayPartsMatch(SignatureHelpItem i1, SignatureHelpItem i2
             private static bool CompareParts(TaggedText p1, TaggedText p2)
                 => p1.ToString() == p2.ToString();
 
-            /// <summary>
-            /// Returns <see langword="null"/> if our work was preempted and we want to return the 
-            /// previous model we've computed.
-            /// </summary>
             private async Task<(ISignatureHelpProvider provider, SignatureHelpItems items)?> ComputeItemsAsync(
```

Comment:
can't return null.  can you make this not be nullable anymore?

Reply:
Done. Thanks